common/php/class-module-with-view.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+
+class EF_Module_With_View extends EF_Module {
+
+
+	/**
+	 * Whether or not the current page is an Edit Flow settings view (either main or module)
+	 * Determination is based on $pagenow, $_GET['page'], and the module's $settings_slug
+	 * If there's no module name specified, it will return true against all Edit Flow settings views
+	 *
+	 * @since 0.8.3
+	 *
+	 * @param string $slug (Optional) Module name to check against
+	 * @return bool true if is module settings view
+	 */
+	public function is_module_settings_view( $slug = false ) {
+		global $pagenow, $edit_flow;
+
+		// All of the settings views are based on admin.php and a $_GET['page'] parameter
+		if ( $pagenow !== 'admin.php' || ! isset( $_GET['page'] ) )
```

Comment:
Braces should be added to match [WordPress PHP Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style)

Reply:
I tried to keep this PR short so I just moved the code around without updating the PHCPS standards support to keep it short. I agree that we should work on this, but let's leave that for a different PR.